systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMaker2ST.java

```diff
@@ -570,6 +648,18 @@ void testMirrorMaker2CorrectlyMirrorsHeaders() {
      * This test is using the Kafka Identity Replication policy. This is what should be used by all new users.
      */
     @ParallelNamespaceTest
+    @TestDoc(
+        description = @Desc("Checks MM2 with IdentityReplicationPolicy, preserving topic names between clusters."),
+        steps = {
+            @Step(value = "Deploy source and target clusters and a scraper pod.", expected = "Test infra is ready."),
```

Comment:
Previous test uses `prerequisites`, this `test infra`, previous ones just clusters....wold be good to unify it